docs/sources/features/datasources/x-ray.md

```diff
@@ -0,0 +1,134 @@
+
+### AWS credentials file
+
+Create a file at `~/.aws/credentials`. That is the `HOME` path for user running grafana-server.
+
+> NOTE: If you think you have the credentials file in the right place but it is still not working then you might try moving your .aws file to '/usr/share/grafana/' and make sure your credentials file has at most 0644 permissions.
+
+Example content:
+
+```bash
+[default]
+aws_access_key_id = <your access key>
+aws_secret_access_key = <your access key>
+region = us-west-2
+```
+
+## The Query Editor 
+
+The most important field in the editor is the query type. There are 4 query types Trace List(Traces in AWS), Trace Statistics, Trace Analytics(Analytics in AWS), Insights.
+
+{{< docs-imagebox img="/img/docs/v72/x-ray-query-editor.png" max-width="800px" class="docs-image--left" caption="X-Ray query editor" >}}
+
+### Trace List
+
+The trace list is pretty much the same as in AWS. Clicking on the trace id in the first column opens the trace on the right side. Notice the query field in the editor which works the same as in AWS. You can write queries, filter expressions or a trace id there.
+
+{{< docs-imagebox img="/img/docs/v72/x-ray-trace-list.png" caption="X-Ray trace list in Grafana" >}}
+
+### Trace Statistics
+
+In trace statistics you can see a graph and a table showing information about error, fault, throttle, success and total count. You can use the columns field in the query editor to only see specified columns.
+
+{{< docs-imagebox img="/img/docs/v72/x-ray-trace-statistics.png" caption="X-Ray trace statistics in Grafana" >}}
+
+### Trace Analytics
+
+In trace analytics you can visualize one of the tables from Analytics in AWS.
+
+### Insights
+
+In insights you can see the summary table for insights just like in AWS. Clicking the InsightId will take you to AWS console.
+
+### Alerting
+
+Since X-Ray queries can return numeric data, alerts are supported. See the [Alerting]({{< relref "../../alerting/alerts-overview.md" >}}) documentation for more on Grafana alerts.
+
+## Pricing
+
+> With AWS X-Ray, there are no upfront fees or commitments. You pay only for what you use, based on the number of traces recorded, retrieved, and scanned. The first 1,000,000 traces retrieved or scanned each month are free. Beyond the free tier, traces scanned cost $0.50 per 1 million traces scanned ($0.0000005 per trace).
+Please see the [X-Ray pricing page](https://aws.amazon.com/xray/pricing/) for more details.
+
+## Configure the data source with provisioning
+
+It's now possible to configure data sources using config files with Grafana's provisioning system. You can read more about how it works and all the settings you can set for data sources on the [provisioning docs page]({{< relref "../../administration/provisioning/#datasources" >}})
+
+Here are some provisioning examples for this data source.
+
+### Using a credentials file
```

Comment:
When would you use the credentials file vs. the other one?

Reply:
It depends on the authentication type. How should I make this clear?

Reply:
After the heading, add an introduction. Something like:

If you are using THIS authentication type, then you should use a credentials file. Otherwise, use OTHER WAY.